### PR TITLE
fix: add CSAIL to companies

### PR DESCRIPTION
**Pull request by digger-yu**
# Add/Fix Dictionary

fix: add CSAIL to companies

## Description

fix: add CSAIL to companies

## References

| CSAIL | Computer Science & Artificial Intelligence Laboratory | https://www.csail.mit.edu/ | 

## Checklist

- [x] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

### CODE REVIEW

File: dictionaries/companies/dict/companies.txt [hunk 1]
```diff
@@ -2078,3 +2078,4 @@ ru
 synology
 undraw
 university
+CSAIL
```
Comment (Jason3S) [inline dictionaries/companies/dict/companies.txt:2081]: Please add it to `dictionaries/companies/src/companies.txt`

**Comment by Jason3S** [comment]
@digger-yu,

I accidently pushed this. Your change will get lost because it was not added to the source.

File: dictionaries/companies/dict/companies.txt (hunk 1 above)
Comment (digger-yu) [reply dictionaries/companies/dict/companies.txt:2081]: in new pull request i fix it 
https://github.com/streetsidesoftware/cspell-dicts/pull/2687

Comment (Jason3S) [reply dictionaries/companies/dict/companies.txt:2081]: Thank you.

**Pull request merged**